### ISSUE TITLE
Fullscreen button is throwing errors

### ISSUE THREAD

**Issue by xavijam** [CartoDB/cartodb.js#412]
[js error](https://my.trackjs.com/details/9af2df365821487993272effc3c308c0)

```
_toggleFullScreen@http://cartodb-libs.global.ssl.fastly.net/cartodbui/assets/3.7.31/javascripts/cdb.js 18:28814  Source
[native code]
dispatch@http://cartodb-libs.global.ssl.fastly.net/cartodbui/assets/3.7.31/javascripts/cdb.js 2:9602  Source
h@http://cartodb-libs.global.ssl.fastly.net/cartodbui/assets/3.7.31/javascripts/cdb.js 2:5517  Source
http@//d2zah9y47r7bi2.cloudfront.net/releases/current/tracker.js 3:497
dispatchEvent@[native code]
_simulateEvent@http://cartodb-libs.global.ssl.fastly.net/cartodbui/assets/3.7.31/javascripts/cdb.js 10:14476  Source
_onUp@http://cartodb-libs.global.ssl.fastly.net/cartodbui/assets/3.7.31/javascripts/cdb.js 10:14054  Source
f@http://cartodb-libs.global.ssl.fastly.net/cartodbui/assets/3.7.31/javascripts/cdb.js 8:29305
```

**Comment by xavijam** [comment]
@alonsogarciapablo is you can check it with him or in your own, perfect too :).

**Comment by Xatpy** [comment]
Sorry guys, but I was ill yesterday and I couldn't check it.

@alonsogarciapablo did you check anything about it?

@xavijam , I've checked that those error is happening with iPhone (Safari Mobile / iOS 8.2). 

Also, could be [this fix](https://github.com/CartoDB/cartodb.js/pull/400) related? I did it last week, but I couldn't test it with iPhone.

**Comment by alonsogarciapablo** [comment]
Hello Xatpy. I missed this yesterday too. Do you need some help?

**Comment by Xatpy** [comment]
I've been trying to replicate it, but I couldn't. 

I've tried with Chrome 41.0.2272.101 / Safari 8.0.2 (10600.2.5) / Firefox (36.0.1) on OSX Yosemite 10.10.1; and Chrome/Android. Also, I did a quick test with iOS7/Safari, but it was quick because it was my girlfriends' phone and she has to go earlier.

@alonsogarciapablo Do you have an iOS 8? If so, could you test it?

@xavijam , The weird thing that I see here is: if it's a mobile, the fullscreen shouldn't be available. I tested it. So, I don't know how the user could click on fullscreen :confused:

**Comment by alonsogarciapablo** [comment]
I don't have iOS 8 :-(

**Comment by Xatpy** [comment]
Trying to test with this [link](http://rtlfr.cartodb.com/viz/5a469992-d135-11e4-abb8-0e0c41326911/embed_map?force_mobile=false) (forcing mobile to false)

**Comment by Xatpy** [comment]
Closed with commit fc91c2bfc2762b71b394777efcb42a3b2075a419
